raster/src/main/scala/geotrellis/raster/RasterExtent.scala

```diff
@@ -67,18 +67,12 @@ case class GeoAttrsError(msg: String) extends Exception(msg)
   * RasterExtent.
   */
 case class RasterExtent(extent: Extent, cellwidth: Double, cellheight: Double, cols: Int, rows: Int)
-    extends GridDefinition {
+    extends GridDefinition with Grid {
```

Comment:
Weird a `GridDefinition` isn't a grid. Is this the best naming? Where a `Grid` can only contain Int cols and rows?